aten/src/ATen/native/native_functions.yaml

```diff
@@ -7289,27 +7289,24 @@
 - func: sort.values(Tensor self, int dim=-1, bool descending=False, *, Tensor(a!) values, Tensor(b!) indices) -> (Tensor(a!) values, Tensor(b!) indices)
   device_check: NoCheck   # TensorIterator
   dispatch:
-    CPU: sort_out_cpu
-    CUDA: sort_out_cuda
+    CPU, CUDA: sort_out
```

Comment:
Technically `sort_out` and `sort` are both composite now, right? Since you implemented them as re-dispatching into the `sort.stable` overload. So you could mark them as `CompositeExplicitAutograd` instead.

Reply:
You are right. I will do it.